training/src/main/java/wolox/training/TrainingApplication.java

```diff
@@ -2,12 +2,16 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
+@EnableJpaRepositories("com.baeldung.persistence.repo")
+@EntityScan("com.baeldung.persistence.model")
```

Comment:
Erase these annotations

Reply:
yeah! how we spoke, I deleted it in this pr [https://github.com/wolox-training/gs-java/pull/5](pr 5)